.gitignore

```diff
@@ -1,3 +1,7 @@
+
```

Comment:
You'll need to re-add the ignore for `bintray_user_auth_secrets.properties` -- that's there very much on purpose!